.github/workflows/pr.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         run: npm run lint
 
       - name: Check package.json
-        run: npm run sort-package-json -- check
+        run: npm run sort-package-json -- --check
```

Comment:
Nice catch.

NPM is making things weird, previously `--` is not needed, already run into this problem several times.